app/views/homes/index.html.erb

```diff
@@ -1,3 +1,3 @@
-<div id="app">
+<div class='grid-x grid-margin-x align-spaced pizzaContainer' id="app">
```

Comment:
this is fine, but might actually recommend keeping it like you had before in the `application.html.erb` file like you had before since other "layout" styling is applied there for every page

Reply:
So we tried having it in "layouts" but due to the amount of div nesting going on we found that it worked better for our project if we put it here in the index.html.